.rubocop_todo.yml

```diff
@@ -381,10 +381,6 @@ Naming/VariableNumber:
 # Configuration parameters: AutoCorrect.
 Performance/HashEachMethods:
   Exclude:
```

Comment:
@Ronaq13 You may want to remove all the lines related to this cop. Since you have already fixed the offenses related to it, these lines have no meaning.
<img width="365" alt="screen shot 2017-12-09 at 8 32 26 pm" src="https://user-images.githubusercontent.com/13951936/33801410-3418e6f4-dd20-11e7-8b8c-b6d5ef37c707.png">